templates/base.html

```diff
@@ -81,7 +81,7 @@
 
 
     {# Stylesheets #}
-    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/KaTeX/0.9.0/katex.min.css" crossorigin="anonymous">
+    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/KaTeX/0.10.0/katex.min.css" crossorigin="anonymous">
```

Comment:
Je viens de réessayer et ça ne fonctionne chez moi que si je fais cette modification, je ne sais pas pourquoi en passant par le CDN ça ne fonctionne pas :/

```suggestion
    <link rel="stylesheet" href="{% static "css/katex.min.css" %}">
```